app/controllers/v1/purls_controller.rb

```diff
@@ -18,7 +18,15 @@ def update
                 retry
               end
 
-      Racecar.produce_sync(value: cocina_object.to_json, key: druid_param, topic: "purl-updates")
+      # Get the actions from the release tags on the cocina model. In the near future,
+      # the releaseTags will be removed from Cocina.
+      actions = { index: [], delete: [] }.tap do |releases|
+        cocina_object.administrative.releaseTags.each do |tag|
+            releases[tag.release ? :index : :delete] << tag.to
```

Comment:
Indentation looks excessive here but I guess if rubocop doesn't care, I'm not too fussed